### PR TITLE
Add API for Retry failed message

### LINKED ISSUE #207: Messaging API - May 2020 update
News: https://developers.line.biz/en/news/2020/05/12/messaging-api-update-may-2020/

- [x] Message character limit and media file usage conditions changed
- [x] [Get LINE emoji information from the text object of a webhook event](https://developers.line.biz/en/reference/messaging-api/#wh-text)
- [ ] [Retry failed message](https://developers.line.biz/en/reference/messaging-api/#retry-api-request)

### PR DESCRIPTION
**Pull request by kkdai**
Fixed #207  
API for Retry failed message 
Refer: https://developers.line.biz/en/reference/messaging-api/#retry-api-request

### CODE REVIEW

**Comment by kkdai** [comment]
@clsung  would you please help to check if you have time?

File: linebot/client.go [hunk 1]
```diff
@@ -170,6 +171,10 @@ func (client *Client) url(base *url.URL, endpoint string) string {
 func (client *Client) do(ctx context.Context, req *http.Request) (*http.Response, error) {
 	req.Header.Set("Authorization", "Bearer "+client.channelToken)
 	req.Header.Set("User-Agent", "LINE-BotSDK-Go/"+version)
+	if len(client.retryKeyID) > 0 {
+		req.Header.Set("X-Line-Retry-Key", client.retryKeyID)
+
```
Comment (clsung) [inline linebot/client.go:176]: redundant newline?

**Review by clsung — APPROVED** [review]
LGTM except the redundant newline

File: linebot/client.go (hunk 1 above)
Comment (kkdai) [reply linebot/client.go]: fixed

**Pull request merged**